python/README.md

```diff
@@ -50,7 +50,49 @@ object_name = 'file1'
 data = sc.textFile(cos.url(object_name, bucket_name))
 ```
 
-### Bluemix / Data Science Experience
+### Bluemix Cloud Object Storage / Data Science Experience
+The class CloudObjectStorage allows you to connect to bluemix cos. You can connect to bluemix using api keys
+as follows:
+
+```pythonw
+import ibmos2spark
+
+# @hidden_cell
+credentials = {
+    'endpoint': 'XXX',
+    'api_key': 'XXX',
+    'service_id': 'XXX'
+}
+
+configuration_name = 'os_bluemix_cos_config'
+cos = ibmos2spark.CloudObjectStorage(sc, credentials, configuration_name, 'bluemix_cos')
+
+bucket_name = 'bucket_name'
+object_name = 'file_name'
+data = sc.textFile(cos.url(object_name, bucket_name))
+```
+
+Alternatively, you can connect to bluemix cos using IAM token. Example:
+```pythonw
+import ibmos2spark
+
+# @hidden_cell
```

Comment:
I think that if we add the `# @hidden_cell` we should 1) make a link to some docs to describe what it is, and 2) make a separate code snippet to encourage developers/users to separate hidden cells from the other parts of the code that doesn't need to be hidden.

Reply:
We don't need that at all actually. It is a DSX thing. I took it out